src/components/user/UpdateUserRoles.tsx

```diff
@@ -30,7 +30,10 @@ export default function UpdateUserRoles(props: { id: number }) {
       roles: newRoles.map((role) => role.id),
     };
 
-    await api('Roles updated successfully!').updateUserRoles(variables);
+    await api(
+      'Roles updated successfully! Any Logged in Users will still have old permissions until they log back in',
```

Comment:
Is the casing for `Any Logged in Users` correct?

Reply:
I think it should be "Any logged in users will still have old permissions until they log back in."